imp_developer.py

```diff
@@ -46,18 +47,22 @@
 from .modules.Sublime_AdvancedNewFile_1_0_0.advanced_new_file.commands import AdvancedNewFileNew
 
 # Generic plugin constants
-PL_BUILD_API_URL_BASE    = "https://build.electricimp.com"
-PL_BUILD_API_URL_V4      = PL_BUILD_API_URL_BASE + "/v4/"
+PL_IMPCENTRAL_API_URL_BASE = "https://api.electricimp.com"
```

Comment:
`=`'s alignment is broken here, doesn't look as pretty as before )

Reply:
fixed indentation